packages/react-instantsearch-hooks/src/__tests__/useSortBy.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import { createInstantSearchTestWrapper } from '../../../../test/utils';
+import { useSortBy } from '../useSortBy';
+
+const getSortByTestItems = [
```

Comment:
We can rename this now. Suggestion:

```suggestion
const items = [
```